zcash_client_sqlite/src/wallet.rs

```diff
@@ -2453,6 +2464,7 @@ pub(crate) fn mark_transparent_utxo_spent(
     tx_ref: i64,
     outpoint: &OutPoint,
 ) -> Result<(), SqliteClientError> {
+    // FIXME: This can select utxos with unknown height (sentinel -1); is that correct?
```

Comment:
I think this is correct, because unmined transparent outputs can be spent (by other unmined transactions), and here we are just tracking spentness.